recipes/libsrtp/all/conanfile.py

```diff
@@ -48,43 +53,85 @@ def requirements(self):
         if self.options.with_openssl:
             self.requires("openssl/[>=1.1 <4]")
 
+        if self.options.with_nss:
+            self.requires("nss/[>=3.93 <4]")
+
+        if self.options.with_mbedtls:
+            self.requires("mbedtls/[>=3.5.1 <4]")
+
```

Comment:
```suggestion
        if self.options.with_nss:
            self.requires("nss/3.93")

        if self.options.with_mbedtls:
            self.requires("mbedtls/3.5.2")

```
Version ranges are only allowed for a small subset of dependencies on CCI currently, unfortunately.